backend/typescript/services/implementations/authService.ts

```diff
@@ -119,10 +119,13 @@ class AuthService implements IAuthService {
         .auth()
         .generatePasswordResetLink(email);
       const emailBody = `
-      Hello,
+      Hey neighbour!
+      <br><br>
+      Thank you for getting involved with mutual aid through Community Fridge KW. We’re thrilled to have you. 
+
       <br><br>
-      We have received a password reset request for your account.
-      Please click the following link to reset it.
+Please confirm that this is the email address (${email}) you would like to use to schedule donations with Community Fridge KW by clicking “Verify”.
+
       <strong>This link is only valid for 1 hour.</strong>
       <br><br>
       <a href=${resetLink}>Reset Password</a>`;
```

Comment:
Isnt this the reset password email? The content talks about verifying email address.

Reply:
it was the styling for an updated verifying email template but we don't have to include it for mvp